src/components/navigation.jsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from 'react';
+import { Link, NavLink } from 'react-router-dom';
+import logo from '../assets/logo.webp';
+import '../styles/navigation.scss';
+
+const Navigation = () => {
+  const [open, setOpen] = useState(true);
+
+  //   This isAdmin state will be fetched from Redux store but for now we will use useState
+  //   const isAdmin = useSelector((state) => state.user.isAdmin);
+  const isAdmin = false;
+
+  return (
+    <nav className="navbar">
+      <div className="nav-container">
+        <div className="nav-group">
+          <div className="nav-header">
+            <button className="navbar-toggler-button" type="button" onClick={() => setOpen(!open)}>
+              <span className="navbar-toggler-icon" />
+            </button>
+            <Link className="nav-logo" to="/">
+              <img src={logo} alt="logo" />
+            </Link>
+          </div>
+          <div className={`collapse navbar-collapse ${open ? 'show' : ''}`}>
+            <ul className="navbar-nav">
+              <li>
+                <NavLink className="nav-link" to="/" end>Home</NavLink>
```

Comment:
Would it make sense to add a left margin to the `nav-link` class (or to the `active` class)? This is to follow the design more closely

![image](https://github.com/DavidEspinoG/mc_videogames_front/assets/70329406/bfaf80cf-4795-483a-970d-70d5eb3691f9)